README.md

```diff
@@ -24,7 +24,7 @@
 - [License](#license) 
 
 ## Install
-`npm install loggaby`
+`npm install loggaby` or `yarn add loggaby`
```

Comment:
it should say "Or with Yarn:" instead of just "or `<yarn install>`"

Reply:
Okay

Reply:
ex) `Or with Yarn: yarn add loggaby`

@TorchedSammy  right?

Reply:
yes, but please newline after `:`

Reply:
`Or with Yarn: 
yarn add loggaby`

Reply:
👌 yup